reaktive/src/commonMain/kotlin/com/badoo/reaktive/completable/Retry.kt

```diff
@@ -0,0 +1,34 @@
+package com.badoo.reaktive.completable
+
+import com.badoo.reaktive.base.operator.Retry
+import com.badoo.reaktive.base.subscribeSafe
+import com.badoo.reaktive.disposable.Disposable
+import com.badoo.reaktive.disposable.DisposableWrapper
+import kotlin.reflect.KClass
+
+fun Completable.retry(predicate: (Int, Throwable) -> Boolean = { _, _ -> true }): Completable =
```

Comment:
Let's add a name for `Int` parameter. Like `predicate: (attempt: Int, Throwable) -> ...`